### ISSUE TITLE
Bug with the all devices option when using deej controller

### ISSUE THREAD

**Issue by rp1231** [applapp/EarTrumpet.HardwareSupportAddon#7]
Hi,
Just wanted to report that there is a bug with the all devices option when using deej controller.
The volume change works fine when I select a specific device.
But when "all devices" is selected, the top title on the overlay flickers quite a bit and the name keeps on cycling between all the
connected audio devices.
The volume change doesn't happen as well.
The app ends up freezing for some time after that.
It seems like it's getting confused as to which device to change the volume for,
Or maybe just getting confused as to which device to display in the overlay.

Thanks 
![EarTrumpet_0HDMpAm3Ef](https://user-images.githubusercontent.com/47380919/153697324-91c5284d-2c57-41f5-8d0a-fbd50eb06564.png)

**Comment by rp1231** [comment]
Also wanted to report that this problem only occurs when the Overlay is activated.
The option works fine when the Overlay is turned off.

**Comment by svenwml** [comment]
Yes, currently the overlay gets triggered for each device's volume change. Since in your configuration, all devices change volumes simulaneously, the overlay is re-triggered for each device which causes the "flickering" / name change. There is better handling required for that in future.

However, I cannot reproduce the problem that the volume change does not happen and that the application freezes.

What you could try if you want to use the overlay and the volume change for all devices is commenting out the code inside the TriggerOSDForDevice method in the file Addon.cs. This will skip overlay displaying for device volume changes but you will still get overlays for application volume changes.

**Comment by rp1231** [comment]
Thanks,
I'll try that out.

Maybe an "active/default device" option in the dropdown menu could be a possible solution?
